ros_buildfarm/templates/status/js/setup.js

```diff
@@ -238,15 +238,24 @@ function filter_table() {
       result_rows = $.map(window.rows, function(row) {
         for (var i = 0; i < queries.length; i++) {
           var is_known_query = false;
+          var is_id_query = false;
           for(var q in QUERY_TRANSFORMS) {
-            if (queries[i] === QUERY_TRANSFORMS[q]) {
+            if (RegExp("^("+QUERY_TRANSFORMS[q]+")$").test(queries[i])) {
               is_known_query = true;
+              if (q === "id") {
```

Comment:
Yeah, I had tried it that way but ran into issues when there were special characters involved
e.g. I wanted [this](https://github.com/ros-infrastructure/ros_buildfarm/pull/279/files#diff-e404e3f34fac28ad7e08164c36763016R38) to link to `?q=id=metapackages` but it would get cut short because of the two `=`s, so I resorted to using dots

FWIW, this is just the name of the query, not the query itself https://github.com/ros-infrastructure/ros_buildfarm/pull/279/files#diff-e404e3f34fac28ad7e08164c36763016R14